governance/proposals/016-migrate-udt-to-base.js

```diff
@@ -0,0 +1,89 @@
+const { ethers } = require('hardhat')
+const {
+  getNetwork,
+  getERC20Contract,
+} = require('@unlock-protocol/hardhat-helpers')
+const l1BridgeAbi = require('../helpers/abi/l1standardbridge.json')
+
+// TODO: change to mainnet / base
+const srcChainId = 11155111 // 1
+const destChainId = 84532 // 8453
```

Comment:
we can remove all mentions of testnets from the proposal if necessary

Reply:
let's do it please nefore you merge!